doc/plugins.md

```diff
@@ -316,6 +316,8 @@ The list of plugins:
     — configure remark w/ YAML
 *   🟢 [`remark-code-title`](https://github.com/kevinzunigacuellar/remark-code-title)
     — add titles to code blocks
+*   🟢 [`remark-hexo`](https://github.com/bennycode/remark-hexo)
+    — renders [Hexo tags](https://hexo.io/docs/tag-plugins)
```

Comment:
Can you sort it in the list? Thanks

(I see the one above is also wrongly places, meh, anyway, feel free to ignore that!)

Reply:
No problem! I sorted mine as well as the one from above (remark-code-title) :)